### DISCUSSION TITLE
Load custom XXfile for configuration

### DISCUSSION THREAD

**Discussion by MartinGonzalez** [fastlane/fastlane#17774] [Q&A] [answered]
Hi! :D
I'm creating a Unity plugin and I would **really** like to standardise a way to configure it just like gym does with Gymfile, in this case a Unityfile where i will have configs like the following

```ruby
android_build({
    bundle_id: 'com.company.product',
    version: '0.0.1'
})

ios_build({
    provisioning_profile: 'com.company.product AdHoc',
    entitlements: [
        "aps-environment" => 'production'
    ],
    info_plist:[
        ITSAppUsesNonExemptEncryption: false
    ]
})
```

How would be the right way to load this file so it can be able to see plugin `android_build` and `ios_build` methods?

Thanks

**Comment by MartinGonzalez** [comment, marked as answer]
I found a way, checking how fastlane does XXfiles loading using eval :D

```ruby
eval(File.read("#{Dir.pwd}/Unityfile"))
```

**Comment by csc-EricWu** [comment]
@MartinGonzalez I think it's necessary for build_ app, resign, update_ info_ Plist add an info_ Plist. Can you do that?
